common-design-patterns/global-header/site-menu.md

```diff
@@ -243,6 +244,10 @@ title: "Theme and topic menu"
 
 <h2 id="changes">Latest changes</h2>
 <dl class="dl-horizontal">
+ <dt>
+    <time datetime="2025-07-08" class="link-muted">2025-07-08</time>
```

Comment:
```suggestion
    <time>2025-07-08</time>
```
The class doesn't do anything and the date is already in ISO format, no need to be more explicit.